test/integration/default/controls/wmi_spec.rb

```diff
@@ -14,7 +14,7 @@
 # Use win32_service with filter, it returns a single service object
 describe wmi({
   class: 'win32_service',
-  filter: "name like '%winrm%'"
+  filter: 'name like "%winrm%"'
```

Comment:
Isn't this breaking existing users that used the old syntax?

Reply:
I'm not sure I understand what you mean. The original test checks if a nested string works with single-quotes (`'`) whereas this test checks against nested double-quotes (`"`).

This small change makes it so the same test covers 2 concepts instead of just one.

Since the resource uses nested `"`'s inside of a string composed with `"`'s, it makes sense to check that the string given here uses `"`'s and that it escapes them properly.

Additionally it checks that the `filter` option actually works as expected.

What's the question here? What concern is there about a breaking change?

Reply:
Seems like it was late during my review. All good, thank you for catching this one!